examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/SubscriptionOptionExtensions.kt

```diff
@@ -9,5 +9,16 @@ fun SubscriptionOption.toButtonString(isDefault: Boolean): String {
         val cycles = pricingPhase.billingCycleCount
         "\t[$formatted, $iso, $cycles cycles]"
     }
-    return "${if (isDefault) "DEFAULT\n" else ""} PricingPhases = [\n$pricingPhasesString\n],\nTags = $tags"
+    val installmentsString = installmentsInfo?.let { installmentsInfo ->
+        val commitmentPaymentsCount = installmentsInfo.commitmentPaymentsCount
+        val subsequentCommitmentPaymentsCount = installmentsInfo.subsequentCommitmentPaymentsCount
+        "\nInstallmentsInfo = [" +
+            "\n\tCommitment: $commitmentPaymentsCount," +
+            "\n\tSubsequent: $subsequentCommitmentPaymentsCount" +
+            "\n]"
+    }
+    return "${if (isDefault) "DEFAULT\n" else ""} " +
+        "PricingPhases = [\n$pricingPhasesString\n],\n" +
+        "Tags = $tags" +
+        if (installmentsString != null) ",$installmentsString" else ""
```

Comment:
Not too pretty, but I think it might be useful: 
<img width="260" alt="image" src="https://github.com/RevenueCat/purchases-android/assets/808417/ed21e508-4b30-4265-82a9-cd5255e6c5db">

Reply:
Definitely useful!

Reply:
❤️  it